### PR TITLE
Feat add noise sounder

### PR DESCRIPTION
**Pull request by obejarano**
I made changes to both the sounder and python tools. I've tested everything several times on falcon/merlin/powder but please take a look at the code, especially the sounder part

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # RENEWLab
 
-[![Build Status](https://falcon.ecg.rice.edu:443/buildStatus/icon?job=github_public_renewlab%2Fci-badge-update)](https://falcon.ecg.rice.edu:443/job/github_public_renewlab/job/ci-badge-update/)
+[![Build Status](https://4489496cb62a.ngrok.io/buildStatus/icon?job=github_public_renewlab%2Ffeat-add-noise-sounder)](https://4489496cb62a.ngrok.io/job/github_public_renewlab/job/feat-add-noise-sounder/)
```
Comment (mindy-min-zhang) [inline README.md:3]:
```suggestion
[![Build Status](https://falcon.ecg.rice.edu:443/buildStatus/icon?job=github_public_renewlab%2Fdevelop)](https://falcon.ecg.rice.edu:443/job/github_public_renewlab/job/develop/)
```

File: CC/Sounder/recorder_worker.cc [hunk 2]
```diff
@@ -230,7 +246,7 @@ herr_t RecorderWorker::initHDF5()
         H5::DataSpace pilot_dataspace(kDsDim, dims_pilot, max_dims_pilot);
         this->file_->createDataSet("/Data/Pilot_Samples",
             H5::PredType::STD_I16BE, pilot_dataspace, this->pilot_prop_);
-        pilot_dataspace.close();
+        // pilot_dataspace.close(); // OBCH FIXME - seems to be done during garbage collection gc() func
```
Comment (asedlmayr) [inline CC/Sounder/recorder_worker.cc:249]: This looks to be ok to remove.  The H5::DataSpace destructor should call close.  As time allows, we should remove all the close() calls on  H5::DataSpace objects or add them to all of them.

File: CC/Sounder/files/renew-conf-two-client-3.6g.json [hunk 3]
```diff
@@ -25,7 +25,7 @@
 
         "Clients" : {
                  "sdr_id" : [
-                     "RF3E000029"
+                     "RF3E000080"
```
Comment (asedlmayr) [inline CC/Sounder/files/renew-conf-two-client-3.6g.json:28]: This is probably unnecessary to check in.

File: CC/Sounder/recorder_worker.cc (hunk 2 above)
Comment (asedlmayr) [inline CC/Sounder/recorder_worker.cc:249]:
```suggestion
```

**Review by asedlmayr — APPROVED** [review]

File: CC/Sounder/files/renew-conf-two-client-3.6g.json (hunk 3 above)
Comment (obejarano) [reply CC/Sounder/files/renew-conf-two-client-3.6g.json:28]: yeah this shouldn't be there

File: CC/Sounder/config.cc [hunk 4]
```diff
@@ -585,6 +588,17 @@ int Config::getClientId(int frame_id, int symbol_id)
     return -1;
 }
 
+int Config::getNSFIndex(int frame_id, int symbol_id)
```
[the remark this answers is not in the document]
Comment (obejarano) [reply CC/Sounder/config.cc]: yeah makes sense. done

**Review by rdoosty — APPROVED** [review]

**Pull request merged**